### PR TITLE
Ability to pull parlai config details from a parlai_internal file

### PR DESCRIPTION
**Pull request by JackUrb**
Basic flow here is that if you have a `parlai_internal.mturk.config` file, you can specify some defaults to use in your opts and in your qualifications. This doesn't break any existing flow.

Additional fixes:
- Cleans up default parameters for qualification names
- Cleans up some underscores and makes soft_block functions safer.
- Makes it so that locale qualifications are only added in the mturk utils if they haven't already been provided beforehand

### CODE REVIEW

File: parlai/mturk/core/mturk_utils.py [hunk 1]
```diff
@@ -276,26 +276,34 @@ def remove_worker_qualification(worker_id, qualification_id,
 def create_hit_type(hit_title, hit_description, hit_keywords, hit_reward,
                     assignment_duration_in_seconds, is_sandbox,
                     qualifications=None,
-                    auto_approve_delay=4*7*24*3600, # default 4 weeks
+                    auto_approve_delay=4*7*24*3600,  # default 4 weeks
```
Comment (emilydinan) [inline parlai/mturk/core/mturk_utils.py]: should we consider making this shorter since people often forget to set it?

Comment (JackUrb) [reply parlai/mturk/core/mturk_utils.py]: Yeah probably, maybe I'll default 1 week instead? Will also update docs

Comment (klshuster) [reply parlai/mturk/core/mturk_utils.py]: +1

File: parlai/mturk/core/mturk_utils.py [hunk 2]
```diff
@@ -276,26 +276,34 @@ def remove_worker_qualification(worker_id, qualification_id,
...
+    local_requirements = []
+    has_locale_qual = False
     if qualifications is not None:
-        localRequirements += qualifications
+        for q in qualifications:
+            if q['QualificationTypeId'] == '00000000000000000071':
+                has_locale_qual
+        local_requirements += qualifications
+
+    if not has_locale_qual:
```
Comment (klshuster) [inline parlai/mturk/core/mturk_utils.py]: ❤️

File: parlai/mturk/core/mturk_utils.py [hunk 3]
```diff
@@ -276,26 +276,34 @@ def remove_worker_qualification(worker_id, qualification_id,
...
-            {'Country': 'CA'},
-            {'Country': 'GB'},
-            {'Country': 'AU'},
-            {'Country': 'NZ'}
-        ],
-        'RequiredToPreview': True
-    }]
+    # If the user hasn't specified a location qualification, we assume to
+    # restrict the HIT to some english-speaking countries.
+    local_requirements = []
```
Comment (klshuster) [inline parlai/mturk/core/mturk_utils.py]: super nit - do we wanna call this `locale_requirements` instead of `local_requirements`?

**Comment by JackUrb** [comment]
Right now I don't have an example `config` file would look like - it can implement either of the depicted methods here though.

**Pull request merged**